rockspec/lua-resty-hmac-ffi-1.0-0.rockspec

```diff
@@ -0,0 +1,24 @@
+package = 'lua-resty-hmac-ffi'
+version = '1.0-0'
+
+source = {
+  url = 'git://github.com/jamesmarlowe/lua-resty-hmac',
+  tag = 'v1.0'
+}
+
+description = {
+  summary = 'Lua driver for making and receiving hmac signed requests',
+  homepage = 'https://github.com/jamesmarlowe/lua-resty-hmac',
+  license = 'BSD-2-Clause License'
+}
+
+dependencies = {
+  'lua == 5.1'
+}
```

Comment:
Also, I'm not sure if it is possible to specify OpenSSL as a dependency in luarocks but I thought it might be worth mentioning here.

Reply:
there is no way to specify the `OpenSSL`.